sentry/interceptor.py

```diff
@@ -35,6 +43,27 @@ async def execute_activity(self, input: ExecuteActivityInput) -> Any:
                 "temporal.workflow.namespace", activity_info.workflow_namespace
             )
             scope.set_tag("temporal.workflow.run_id", activity_info.workflow_run_id)
+
+            # Build workflow URL for easy navigation from Sentry
```

Comment:
This is meant to be a sample a user can adjust with these more advanced needs rather than a library. I am not sure we want the sample assuming anything about cloud, though it would totally make sense for some users to set this tag (or any other tag) when they copy this sample and adjust it for their needs.